docs_app/content/guide/observable.md

```diff
@@ -267,7 +267,7 @@ Conclusion:
 
 ## Anatomy of an Observable
 
-Observables are **created** using `Observable.create` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer and Subscription.
+Observables are **created** using `new Observable` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer and Subscription.
```

Comment:
note sure the semantics of this sentence. Might be worth breaking it up after 'creation operator. The sentence starts with are and then after the comma continues with are. Looks to be a run on sentence of sorts

Reply:
I think that the first part of the sentence, 'These four aspects....' is important, so when 'but' is used, it is negating that first part.

Reply:
Will it make it easier to understand if was something like: 

> There are four important aspects when we talk about an Observable instance.
> 
> Observables:
>   * are **created** using `new Observable` or a creation operator.
>   * are **subscribed** to with an Observer.
>   * **execute** to deliver `next` / `error` / `complete` notifications to the Observer.
>   * may be **disposed**.
>
> While all of these aspects are encoded in an Observable instance, some of them are also related to other types, like Observer and Subscription.

Reply:
@pertrai1 any comments on my suggestions above? :)